client/src/modules/Sidebar/components/ItemList/components/DefItem/EditDefItem/date.js

```diff
@@ -0,0 +1,10 @@
+const dateTime = () => {
+  const d = new Date();
+  const year = d.getFullYear();
+  const month = (`0${  d.getMonth() + 1}`).slice(-2);
+  const day = (`0${  d.getDate()}`).slice(-2);
+  const datestring = `${year  }-${  month  }-${  day}`;
+  return datestring;
+};
+
+export default dateTime;
```

Comment:
You can replace it with one row - new Date().toISOString().split('T')[0]. Result will be 'yyyy-mm-dd'